src/main/java/com/manywho/services/pdf/actions/ConcatenatePdfCommand.java

```diff
@@ -29,7 +32,10 @@ public ConcatenatePdfCommand(FileManager fileManager) {
                 throw new RuntimeException("Concatenate PDF with owner password restrictions is not supported.", e);
             }
 
-            throw new RuntimeException(e.getMessage(), e);
+            String error = String.format("There was a problem concatenating the PDF from the provided files: %s", e.getMessage());
+            LOGGER.error(error, e);
```

Comment:
The message passed into the logger shouldn't have the message, as it'll just duplicate what's in the stack trace already. Also, if you use parameters in a log message, you should use `%s` or %d` (or whatever format) in the message, and pass in the variables as varargs, rather than prerendering the message.

That way, our logging handler will group all of the errors by format template rather than having a million slightly-different individual rendered messages.